src/test_workflow/test_recorder/test_recorder.py

```diff
@@ -95,10 +95,12 @@ class LocalClusterLogs(LogRecorder):
 
     def __init__(self, parent_class: TestRecorder) -> None:
         self.parent_class = parent_class
+        self.number_of_nodes = 0
 
     def save_test_result_data(self, test_result_data: TestResultData) -> None:
         base = self.parent_class._create_base_folder_structure(test_result_data.component_name, test_result_data.component_test_config)
-        dest_directory = os.path.join(base, "local-cluster-logs")
+        dest_directory = os.path.join(base, "local-cluster-logs/id-" + str(self.number_of_nodes))
+        self.number_of_nodes += 1
```

Comment:
This seems to be a generic change. Can you make sure that this doesn't impact other plugins? CC: @peterzhuamazon @gaiksaya

Reply:
I think it might impact the resports workflows that @zelinh was working on.

Need confirmation on how he retrieve the logs.

Thanks.

Reply:
Yeah, I was worried this might impact the logs path for component yml file we generated during test record; but i just checked and confirmed that we are walking through the test folder [here](https://github.com/opensearch-project/opensearch-build/blob/318450ec8f4c8a3596321b8ae12adacb81e804c2/src/test_workflow/test_recorder/test_recorder.py#L79-L83). The inside file/directory structure should also be recorded properly. 
I also checked that we aren't hardcorded anywhere `local-cluster-logs` so I think this should be fine.